tests/testdata/buf.build/community/mercari-grpc-federation/v0.10.0/grpc-federation/plugin.sum

```diff
@@ -1 +1 @@
-h1:9qs44J7t2kEDynu+mok5OPLjVRaXKzxHm5O7WXToC4U=
+h1:v/cVrYNOv6AakvnwUF2MrlEL3v80BOnlZrKC2z2MpHg=
```

Comment:
Existing will changed b/e directory layout is different when feeding the paths=source_relative option